_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+                <input type="radio" class="field-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_02" value="Email messages" aria-describedby="updates_email_02_helper">
+                <span class="field-input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="field-hint" id="updates_email_02_helper">We will never spam you. Pinky swear.</div>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_email_04">
+                <input type="radio" class="field-input input-radio replace-radio" name="updates_notifications_02" id="updates_email_04" value="Email messages" disabled>
+                <span class="field-input-label">Never contact me (disabled radio button)</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Checkboxes</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">Select your favorite pizza toppings (default checkboxes)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_01">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_01" value="Pepperoni" checked="checked">
+                <span class="field-input-label">Pepperoni</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_02">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_02" value="Sausage">
+                <span class="field-input-label">Sausage</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_03">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_03" value="Cheese" checked="checked">
+                <span class="field-input-label">Cheese</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_04">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_04" value="Sauce" disabled>
+                <span class="field-input-label">Sauce (disabled checkbox)</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">What is your favorite cereal? (enhanced checkboxes)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_05">
+                <input type="checkbox" class="field-input input-checkbox replace-checkbox" name="checkboxes_02" id="checkbox_05" value="Cinnamon Toast Crunch" checked="checked">
```

Comment:
NIT: Let's keep this consistent with our attribute standards (and what you're doing with required fields/attrs) by just using the ``checked`` notation here. Cool?